src/Infrastructure/LeanCode.ConfigCat/ConfigCatToMSLoggerAdapter.cs

```diff
@@ -88,5 +91,20 @@ public LogValues(ref FormattableLogMessage message)
         IEnumerator IEnumerable.GetEnumerator() => GetEnumerator();
 
         public override string ToString() => Message.InvariantFormattedMessage;
+
+        private string RenderTemplate()
+        {
+            // Message.Format contains template in a format: 'A sample error message with parameter value: {0}'
+            // Serilog does not treat it as valid argument, and logs this as is.
+            // So instead we render the format to get: 'A sample error message with parameter value: {Value}'
+            // Which can be consumed correctly later
+            return Message.ArgNames is not { } argNames
```

Comment:
Client is nullable aware, so I think we can spare this null check, also the `is not` form seems extra confusing for me.

Reply:
`Message` is a `struct` so whenever someone does `= default;` it immediately breaks nullables :disappointed:

I'd prefer to keep this because I'm a bit too paranoid but I could invert the check to hopefully make it more readable.